ci/docker.sh

```diff
@@ -2,7 +2,8 @@
 set -eEuo pipefail
 
 workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
-docker_image_name="${DOCKER_IMAGE_NAME:-evmapp}"
+evmapp_docker_image_name="${EVMAPP_DOCKER_IMAGE_NAME:-evmapp}"
+bootstraptool_docker_image_name="${BOOTSTRAPTOOL_DOCKER_IMAGE_NAME:-bootstraptool}"
 docker_hub_org='horizenlabs'
```

Comment:
Lets change this to 'zencash'. Different credentials might be needed in travis-ci.